### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by opie4624**
Fixed a couple typos.
Updated the build instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,23 +13,30 @@ It's worth mentioning that Agent has two builds.
...
 
 ## Building the electron application
 
-First up, make sure that node >=8.1.x and npm >=5.1.x are installed on your system. Next up:
+### Step 1: Build Dependencies
 
-```
-# Execute the following line on Linux. Use relevant package manager and package names on non-Debian based distros.
-apt-get install libusb-dev libudev-dev g++
+You'll need Node (>=8.1.x) and NPM (>=5.1.x). Use your OS package manager to install these. [Check the NodeJS site for more info.](https://nodejs.org/en/download/package-manager/ "Installing Node.js via package manager") Mac OS users can simply `brew install node` to get both.
```
Comment (ert78gb) [inline README.md]: I suggest to use NPM >= 5.6.0. it is handle correct the os specific optional dependencies in package-lock.json file. Or we just write use the NodeJs LTS :)

Comment (opie4624) [reply README.md]: This isn't a content edit, just a reorganization. Those are the same version constraints already present. Perhaps a follow-on edit updating the version constraints?

**Comment by mondalaci** [comment]
@opie4624 The updated readme looks great to me. Thanks for the edit!

@ert78gb Feel free to edit it further.

File: README.md (hunk 1 above)
Comment (ert78gb) [reply README.md]: I think you could rewrite the content. The first sentence: `You'll need latest NodeJs LTS`. It is 8.9.4 and it is contains npm 5.6.0.
If you would like to add any useful information that help others feel free.

**Pull request merged**